core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -102,3 +103,9 @@ case object KAFKA_0_10_0_IV0 extends ApiVersion {
   val messageFormatVersion: Byte = Message.MagicValue_V1
   val id: Int = 4
 }
+
+case object KAFKA_0_10_0_IV1 extends ApiVersion {
+  val version: String = "0.10.0-IV1"
+  val messageFormatVersion: Byte = Message.MagicValue_V1
+  val id: Int = 4
```

Comment:
I like Gwen's suggestion about using `KAFKA_0_10_0 ` here as we know that will be the final version for the release, but let's see what @junrao says.